gemd/json/gemd_json.py

```diff
@@ -51,21 +24,23 @@ class GEMDJson(object):
     scope: defines the scope to use for autogenerated UUIDs for objects without uids
     """
 
+    _modules = [
+        gemd.entity.attribute,
+        gemd.entity.bounds,
+        gemd.entity.file_link,
+        gemd.entity.link_by_uid,
+        gemd.entity.object,
+        gemd.entity.source,
+        gemd.entity.template,
+        gemd.entity.value
+    ]
     _clazzes = [
-        MaterialTemplate, MeasurementTemplate, ProcessTemplate,
-        MaterialSpec, MeasurementSpec, ProcessSpec, IngredientSpec,
-        ProcessRun, MaterialRun, MeasurementRun, IngredientRun,
-        Property, Condition, Parameter, PropertyAndConditions,
-        PropertyTemplate, ConditionTemplate, ParameterTemplate,
-        RealBounds, IntegerBounds, CategoricalBounds, CompositionBounds,
-        MolecularStructureBounds,
-        NominalComposition, EmpiricalFormula,
-        NominalReal, UniformReal, NormalReal, DiscreteCategorical, NominalCategorical,
-        UniformInteger, NominalInteger, Smiles, InChI,
-        FileLink, PerformedSource
+        obj for mod in _modules for _, obj in inspect.getmembers(mod)
```

Comment:
I'm not a huge fan of making this implicit: if someone forgets to add a class here we should see it in a test.  But if we do make it implicit, I think it should be heavily documented, e.g. in-line and in the contributing guide and in the serialization docs [here](https://github.com/CitrineInformatics/gemd-python/blob/master/docs/source/depth/serialization.rst).